stellarphot/gui_tools/photometry_widget_functions.py

```diff
@@ -68,6 +84,20 @@ def aperture_radius(self):
         """
         return self._aperture_radius
 
+    @property
+    def inner_annulus(self):
```

Comment:
This is fine for now, but I'll be pushing up an aperture settings object in a bit.

Reply:
OK